### DISCUSSION TITLE
Help updating GZdoom to 4.10.0

### DISCUSSION THREAD

**Discussion by SpidFightFR** [void-linux/void-packages#44756] [Q&A] [answered]
Hey guys how are you all doing?
I need help regarding [the PR i tried to make earlier](https://github.com/void-linux/void-packages/pull/44504), basically, the package builds without issue on x64 GlibC, but not on i686 or x64 musl (check [here](https://github.com/void-linux/void-packages/pull/44504/checks))...

At first i thought it was the [patch](https://gist.github.com/SpidFightFR/78914fe5db2062cf247362b91715e21c#file-fix-i686-modified-patch) i made for i686 but after reviewing it, i have trouble finding where the issue lies...

Also there is a [Gist](https://gist.github.com/SpidFightFR/78914fe5db2062cf247362b91715e21c) with the updated template, the patch and some useful links in an index.
Any help is welcome, don't hesitate to propose changes, and i'll make them !

Thanks by advance !

**Comment by classabbyamp** [comment, marked as answer]
continuing in PR
